fieldsolver/gridGlue.cpp

```diff
@@ -388,10 +388,10 @@ void getFieldsFromFsGrid(
          //loop over dccrg cells to which we shall send data for this remoteRank
          auto const &fsgridCells = onFsgridMapCells[dccrgCell];
          for (auto const fsgridCell: fsgridCells){
-         //loop over fsgrid cells for which we compute the average that is sent to dccrgCell on rank remoteRank
-//        if(technicalGrid.get(fsgridCell)->sysBoundaryFlag == sysboundarytype::DO_NOT_COMPUTE) {
-//           continue;
-//        }
+            //loop over fsgrid cells for which we compute the average that is sent to dccrgCell on rank remoteRank
+            if(technicalGrid.get(fsgridCell)->sysBoundaryFlag == sysboundarytype::BOUNDARY_PADDING) {
```

Comment:
Well this is exactly wrong, isn't it? You want to include padding, exclude `DO_NOT_COMPUTE`.

Reply:
That depends on whether we decide the PADDING type to be the one at the ionosphere or the one at the outer boundaries. I understood that PADDING is at the outer boundaries. And sysboundary.cpp L639 Urs' version puts PADDING at the outer boundaries.